automated/linux/ota-update/static-delta.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (C) 2023 Foundries.io Ltd.
+
+# shellcheck disable=SC1091
+. ./sh-lib
```

Comment:
shouldn't this be '. ../../lib/sh-test-lib' ?

Reply:
yes, should be. Let me rebase this branch.